Makefile

```diff
@@ -53,7 +53,7 @@ run-local:
 	operator-sdk up local --operator-flags -webhook=false
 
 login:
-	@docker login -u "$(DOCKER_USER)" -p "$(DOCKER_PASS)"
+	@docker login -u "$(DOCKER_TEST_USER)" -p "$(DOCKER__TEST_PASS)"
```

Comment:
How does this name change help?

Reply:
This doesn't, I just use a different name to avoid deleting your DOCKER_USER and DOCKER_PASS in the Travis setting.